www/kask05/cv01/index.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+$logo = 'logo.png';
+$name = 'Burn underneath the rain';
+$age = 2023;
+$position = 'Band';
+$firm = 'Metalcore';
+$street = 'Plynární';
+$streetnum1 = 1096;
+$streetnum2 = 23;
+$city = 'Prague';
+$phonenum = +420735845678;

```

Comment:
to plusko se vam zobrazuje? telefonni cisla definovat jako string, ne jako cislo